### ISSUE TITLE
Default script testing wrong "invariantname" strings.

### ISSUE THREAD

**Issue by parduz** [EDCD/EDDI#2293]
In the Mission Faction State script, there's a test for the `mission.FactionState.invariantName` variable:
`|elif state = "CivilUnrest":`
But the variable value is "Civil Unrest" with a space.

It would appear that the invariant variable has been updated to have spaces between the words, but the default script not updated to match.

I think this happens also with other values but i've not tested them all.

**Comment by Darkcyde13** [comment]
I can confirm this.  I tried a mission from a faction that is in the 'Civil Unrest' state.  I added this line to the end of the Mission faction state script:
`Faction state: "{mission.FactionState.invariantName}".`
The 'unrest' speech was not voiced from either the accept, or complete, missions scripts, and the additional line voiced this:
`Faction state: "Civil Unrest"`

As Parduz says, it seems the invariant variable has been changed to have a space between words.  While I've only tried the unrest state, I assume it will also apply to the `Civil Liberty` and `Civil War` states as well.

Edit: Just tried 'Civil War', and that is indeed the same.
